client/packages/sdk/src/price-estimation/eth.ts

```diff
@@ -0,0 +1,56 @@
+import fetch from "node-fetch";
+
+// Gas amount for actions
+export const ETH_TRANSFER_GAS_AMOUNT = 21000;
+
+export type EthSpeedMode = "rapid" | "fast" | "standard" | "slow";
+export const EthSpeedModes = {
+  Rapid: "rapid",
+  Fast: "fast",
+  Standard: "standard",
+  Slow: "slow",
+} as const;
+
+export type EthAction = "tran";
+export const EthActions = {
+  Transfer: "tran",
+} as const;
+
+export type GasPrice = {
+  rapid: number;
+  fast: number;
+  standard: number;
+  slow: number;
+  timestamp: number;
+  priceUsd: number;
+};
+
+export const getGasPrice = async () => {
+  const endpoint = "https://beaconcha.in/api/v1/execution/gasnow";
```

Comment:
maybe select for 2 gateways linked to 2 different APIs for Ethereum + Sepolia?

Reply:
Fixed, see commit [9825cee](https://github.com/t3rn/t3rn/pull/1091/commits/9825ceeba5c279173329a3755d4e4d83bb331890).